maestro/lifecycle.py

```diff
@@ -65,10 +65,7 @@ def __repr__(self):
 
     def _test(self, container=None):
         try:
-            s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
-            s.settimeout(1)
-            s.connect((self.host, self.port))
-            s.close()
+            create_connection((self.host, self.port), 1)
```

Comment:
I prefer to keep `socket.create_connection` to make it obvious that this comes from the `socket` module and is not some local method.